src/OpenAIGym.jl

```diff
@@ -19,14 +16,15 @@ const _py_envs = Dict{String,Any}()
 
 # --------------------------------------------------------------
 
-abstract AbstractGymEnv <: AbstractEnvironment
+abstract type AbstractGymEnv <: AbstractEnvironment end
 
 "A simple wrapper around the OpenAI gym environments to add to the Reinforce framework"
-type GymEnv <: AbstractGymEnv
+mutable struct GymEnv <: AbstractGymEnv
     name::String
     pyenv  # the python "env" object
     state
     reward::Float64
+    total_reward::Float64
```

Comment:
I think we should make them parametric in another PR.
e.g.
```julia
mutable struct GymEnv{T<:AbstracFloat}
  ...
  reward::T
  total_reward::T
  ...
```

Reply:
I think the reward being Float64 is probably not such a big issue. The state type being parametric I have in #11

But yeah it probably doesn't hurt

Reply:
:ok_hand: